pkgs/top-level/all-packages.nix

```diff
@@ -17409,7 +17409,7 @@ in
 
   mkvtoolnix = libsForQt5.callPackage ../applications/video/mkvtoolnix { };
 
-  mkvtoolnix-cli = callPackage ../applications/video/mkvtoolnix {
+  mkvtoolnix-cli = mkvtoolnix.override {
```

Comment:
I always wondered: Is there a difference here between `callPackage` and `override` and if yes which should be preferred?

Reply:
override overrides another package. callPackages is creating a package. Doesn't really make any difference except if you have inputs that changed in the original packages and you want to keep them.